website/content/docs/app-config/dynamic.mdx

```diff
@@ -28,13 +28,18 @@ not thoroughly explain the config setting CLI or `waypoint.hcl` stanzas.
 The following is a list of currently available configuration sources
 and their associated documentation:
 
-- [Kubernetes](/plugins/kubernetes#kubernetes-configsourcer) - Read values from ConfigMaps and Secrets.
+- [Terraform Cloud](/plugins/terraform-cloud#terraform-cloud-configsourcer) -
+  Read Terraform state outputs from Terraform Cloud.
+- [Kubernetes](/plugins/kubernetes#kubernetes-configsourcer) - Read values from
+  ConfigMaps and Secrets.
 - [Vault](/plugins/vault#vault-configsourcer) - Read values from Vault secrets,
   including dynamic secrets such as database credentials.
+- [AWS SSM](/plugins/aws-ssm#aws-ssm-configsourcer) - Read configuration values
```

Comment:
Just a suggestion to round out the list! 😄 

```suggestion
- [AWS SSM](/plugins/aws-ssm#aws-ssm-configsourcer) - Read configuration values
- [Consul](/plugins/consul#consul-configsourcer) - Read values from Consul key value storage.
```

Reply:
Love it, but shall we make that a separate PR to `main`? Once this change is in, we can backport as normal.

Reply:
Ah, yes I agree that could be a separate PR!